[test] execute `go test go test -check.f PouchNetworkSuite.TestNetworkConnect` fails

### Ⅰ. Issue Description

When I read the testing document in https://github.com/alibaba/pouch/blob/master/docs/test/test.md, I know how to test integration test.

While when I execute `go test go test -check.f PouchNetworkSuite.TestNetworkConnect`, it fails with the error message:
```shell
root@ubuntu:~/go/src/github.com/alibaba/pouch/test# go test -check.f PouchNetworkSuite.TestNetworkConnect
# github.com/alibaba/pouch/test
cli_network_test.go:15:2: cannot find package "github.com/vishvananda/netlink" in any of:
	/root/go/src/github.com/alibaba/pouch/vendor/github.com/vishvananda/netlink (vendor tree)
	/usr/local/go/src/github.com/vishvananda/netlink (from $GOROOT)
	/root/go/src/github.com/vishvananda/netlink (from $GOPATH)
FAIL	github.com/alibaba/pouch/test [setup failed]
```

I am afraid the guidance in the document does not cover every case, since for some cases GOPATH needs to be set more paths.


### Ⅱ. Describe what happened

CI fails to execute since GOPATH issue.



### Ⅲ. Describe what you expected to happen

Test Document could explicitly cover all the cases to guide users.


### Ⅳ. How to reproduce it (as minimally and precisely as possible)

none

### Ⅴ. Anything else we need to know?


### Ⅵ. Environment:

- pouch version (use `pouch version`):
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

--- a/docs/test/test.md
+++ b/docs/test/test.md
@@ -88,12 +88,23 @@ GOROOT=/usr/local/go
 GOPATH=/go
 ```
 
-Then copy or clone or link pouch source code to the GOPATH：
+Then copy or clone pouch source code to the GOPATH：
 
 ```
 # pwd
 /go/src/github.com/alibaba/pouch
 ```
+
+The following steps are also needed to make sure libnetwork package could be found:
+
+```
+BUILDPATH=/tmp/pouchbuild
+mkdir -p $BUILDPATH/src/github.com/docker
+cp -r /go/src/github.com/alibaba/pouch/extra/libnetwork $BUILDPATH/src/github.com/docker/libnetwork
+export GOPATH=$GOPATH:$BUILDPATH:$BUILDPATH/src/github.com/docker/libnetwork/Godeps/_workspace
+```
+
+And please notice that files in `/tmp` directory may be deleted after reboot.
 
 Now you could run unit test as following:
 

